File: go/docmost/models/search_suggestions.go
package models

// SearchSuggestions from OpenAPI.
type SearchSuggestions struct {
	Users []map[string]any `json:"users,omitempty"`
	Groups []map[string]any `json:"groups,omitempty"`
	Pages []map[string]any `json:"pages,omitempty"`
}

File: c#/src/Docmost.Sdk/Models/SearchSuggestions.cs
namespace Docmost.Sdk.Models;

public class SearchSuggestions
{
    [System.Text.Json.Serialization.JsonPropertyName("users")]
    public List<Dictionary<string, object?>>? Users { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("groups")]
    public List<Dictionary<string, object?>>? Groups { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("pages")]
    public List<Dictionary<string, object?>>? Pages { get; set; }

}
